lib/game_box/arena.ex

```diff
@@ -124,15 +123,15 @@ defmodule GameBox.Arena do
 
     response = Extism.Plugin.call(plugin, "handle_event", Jason.encode!(argument))
 
-    {:reply, response, arena, {:continue, :broadcast}}
+    {:reply, response, arena}
   end
 
-  @impl true
-  def handle_continue(:broadcast, state) do
-    %{arena_id: arena_id, version: version} = state
-    PubSub.broadcast(GameBox.PubSub, "arena:#{arena_id}", {:version, version})
-    {:noreply, state}
-  end
+  # @impl true
```

Comment:
I don't think this is needed because the state shouldn't actually change if we aren't tracking version in elixir